File: Java/IntroJava/02.Operators and Expressions/src/TrapezoidArea.java
/*
 * Write an expression that calculates trapezoid's area by given 
 * sides a and b and height h.
 * /2.0 за да стане double резултата
*/
import java.util.Scanner;
public class TrapezoidArea {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int a = input.nextInt();
        int b = input.nextInt();
        int h = input.nextInt();
        input.close();
        System.out.println("Trapezoid's area is :" + ((a + b) * h / 2.0));
	}

}

File: CSharp/Unit1/02.Operators and Expressions/08.TrapezoidArea/TrapezoidArea.cs
/*
 * Write an expression that calculates trapezoid's area by given 
 * sides a and b and height h.
 * /2.0 за да стане double резултата
*/
using System;
class TrapezoidArea
{
    static void Main()
    {
        int a = int.Parse(Console.ReadLine());
        int b = int.Parse(Console.ReadLine());
        int h = int.Parse(Console.ReadLine());
        Console.WriteLine("Trapezoid's area is :" + ((a + b) * h / 2.0));
    }
}
